### PR TITLE
Moved test from StartView to AdminDashBoard

### LINKED ISSUE #781: Fix tests to #775
:shipit:
Move the LoanListComponent exists in view from 'StartView.spec.ts' to 'AdminDashboardView.spec.ts'

### PR DESCRIPTION
**Pull request by ntijoh-nikki-andersson**
## ⚙️ What issue did you implement or fix?
Fixes #781 

## 📜 Summary
- Fixed tests to #775 

## 📝 Type of change

- [x] 🐛 Bug fix (non-breaking change which fixes an issue)

## ✅ Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## 💩 How can this implementation be improved?
- Change bar into zar. :shipit:

### CODE REVIEW

File: frontend/src/views/__tests__/AdminDashboardView.spec.ts [hunk 1]
```diff
@@ -10,4 +10,10 @@ describe('AdminDashboardView.vue', () => {
     expect(wrapper.find('h1').exists()).toBe(true);
   });
 
+  it('LoanListComponent exists in view', () => {
+    const injectRouterStubs = {stubs: ['router-link']};
+    const wrapper= shallowFactory(StartView as VueClass<Vue>, injectRouterStubs);
```
Comment (Linkaan) [inline frontend/src/views/__tests__/AdminDashboardView.spec.ts:15]:
```suggestion
    const wrapper= shallowFactory(AdminDashboardView as VueClass<Vue>, injectRouterStubs);
```

Comment (ntijoh-nikki-andersson) [reply frontend/src/views/__tests__/AdminDashboardView.spec.ts]: Yeah, forgot to add my change after git hook-thingy

**Comment by Linkaan** [comment]
Great, now please enable branch protection to discourage merging PR with broken tests: https://stackoverflow.com/a/58655352 :)

**Review by ntijoh-Erik-Albinsson — APPROVED** [review]
veri good

**Review by olantig — APPROVED** [review]

**Pull request merged**